pom.xml

```diff
@@ -117,6 +116,23 @@
             <version>RELEASE</version>
             <scope>test</scope>
         </dependency>
+        <dependency>
+            <groupId>junit</groupId>
+            <artifactId>junit</artifactId>
+            <version>4.13</version>
```

Comment:
Remember about maven good practices to extract versions to separate properties.
This way you would  have a chance to notice that you duplicated `junit` dependency in your pom.xml